### PR TITLE
Add 10-in-1: Arcade Collection

### PR DESCRIPTION
**Pull request by kisame78**
Adds 10-in-1: Arcade Collection.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/10-in-1-arcade-collection/game.dat [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title = "10-in-1: Arcade Collection"
+description = "Compilation arcade game\n\nGamelion Studios"
```
Comment (Leo626) [inline games/10-in-1-arcade-collection/game.dat]: A better description would be like this.

`10-in-1: Arcade Collection, known as \"9-in-1: Arcade Collection\" in Japan, is a collection of mini games developed and published by Gamelion Studios. It is comprised of games inspired by classic arcade titles along with a some original ones.`

File: games/10-in-1-arcade-collection/game.dat [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+needs_system_files = false
+needs_shared_font = false
+game_type = "eshop"
+
+[[releases]]
+title = "00040000000E7300"
+region = "eur"
+release_date = "2013-08-15"
```
Comment (Leo626) [inline games/10-in-1-arcade-collection/game.dat]: This date is wrong. Correct date is `2013-08-01`.

File: games/10-in-1-arcade-collection/game.dat [hunk 3]
```diff
@@ -0,0 +1,10 @@
+title = "10-in-1: Arcade Collection"
+description = "Compilation arcade game\n\nGamelion Studios"
+needs_system_files = false
+needs_shared_font = false
+game_type = "eshop"
+
+[[releases]]
```
Comment (DPS2004) [inline games/10-in-1-arcade-collection/game.dat]: there is a NA release too, release date is Aug 15, 2013, and TitleID is 00040000000C1400

**Comment by CodingKoopa** [comment]
Please address the review comments, or this will be closed.

**Comment by CodingKoopa** [comment]
Closing due to inactivity.

**Pull request closed without merging**